pathway-analysis/03-gene_set_variation_analysis.Rmd

```diff
@@ -176,18 +186,19 @@ all_genes <- rownames(rnaseq_mat)
 set.seed(2020)
 ```
 
-Our minimum gene set size earlier was 15 genes and our maximum gene set size was 500 genes. We'll use the same minimum and maximum values for our random gene sets and some values in between.
+Our minimum gene set size earlier was 15 genes and our maximum gene set size was 500 genes. 
+We'll use the same minimum and maximum values for our random gene sets and some values in between.
 
 ```{r gene_set_sizes}
 # Make a list of integers that indicate the random gene set sizes
 gene_set_size <- list(15, 25, 50, 100, 250, 500)
 ```
 
-For each gene set size, we will generate 100 random gene sets
+For each gene set size, we will generate 100 random gene sets.
 
 ```{r random_gene_sets}
 # Set number of replicates
-nreps <-  100
+nreps <- 100
 # Generate 100 random gene sets of each size
 random_gene_sets <- 
   purrr::map(
```

Comment:
Since your comment had be looking back at this, I had a thought for revision. I think it can be confusing to mix `f(x, a)` and `x %>% f(a)`, especially when explaining a new function. We might want to change all of the `purrr::map` functions to follow the latter mode. I also think it is potentially clarifying here, where it will separate the gene size vector creation and the sampling.

For example, I can see people getting hung up with the following block (line 219ish) 
```r
lengths_vector <- purrr::map(random_gene_sets, ~ length(.x)) %>%
  purrr::map(~ paste0("pathway_", .x)) %>%
  # Return a vector
  purrr::flatten_chr()
```

where I think the following might be more easily interpreted:
```r
lengths_vector <- random_gene_sets %>%
  purrr::map(~ length(.x)) %>% 
  purrr::map(~ paste0("pathway_", .x)) %>%
  # Return a vector
  purrr::flatten_chr()
```

I _want_ to show that you can use it either way, but I worry about the cognitive load.